src/community/advanced-data-card.tsx

```diff
@@ -0,0 +1,462 @@
+                            )}
+                            <Text4
+                                color={vars.colors.textPrimary}
+                                truncate={titleLinesMax}
+                                weight="regular"
+                                as={titleAs}
+                                hyphens="auto"
+                            >
+                                {title}
+                            </Text4>
+                            <Text2
+                                color={vars.colors.textPrimary}
+                                truncate={subtitleLinesMax}
+                                as="p"
+                                regular
+                                hyphens="auto"
+                            >
+                                {subtitle}
+                            </Text2>
+                        </Stack>
+                    </Stack>
+                </header>
+
+                {description && (
+                    <Text2
+                        truncate={descriptionLinesMax}
+                        as="p"
+                        regular
+                        color={vars.colors.textSecondary}
+                        hyphens="auto"
+                    >
+                        {description}
+                    </Text2>
+                )}
+            </Stack>
+        </>
+    );
+};
+
+type CardFooterProps = {
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    footerImage?: RendersNullableElement<typeof Image>;
+    footerText?: string;
+    footerTextLinesMax?: number;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+};
+
+const CardFooter: React.FC<CardFooterProps> = ({
+    button,
+    footerImage,
+    footerText,
+    footerTextLinesMax,
+    buttonLink,
+}) => {
+    const hasButton = !!button;
+    const hasFooterImage = !!footerImage;
+    const hasFooterText = !!footerText;
+    const hasButtonLink = !!buttonLink;
+    const hasAllItens = hasButton && (hasFooterImage || hasFooterText) && hasButtonLink;
+
+    const maxWidth = hasButtonLink && !hasAllItens ? '178px' : '';
+
+    return (
+        <>
+            <div className={styles.adjustmentDivider}>
+                <Divider />
+            </div>
+
+            <div
+                className={classNames(
+                    styles.actions,
+                    styles.actionsVariants[hasAllItens ? 'mobile' : 'default']
+                )}
+            >
+                {hasButton && (
+                    <div
+                        tabIndex={-1}
+                        className={classNames(
+                            sprinkles({
+                                display: 'flex',
+                            }),
+                            styles.marginTopButton
+                        )}
+                        style={{
+                            marginRight: 16,
+                            position: 'relative',
+                        }}
```

Comment:
use classes
why position relative?

Reply:
As the position relative, didn't change the design once I removed it, it looks like it's a leftover of some old logic from this component